packages/@sanity/cli/src/actions/init-project/bootstrapLocalTemplate.ts

```diff
@@ -119,29 +122,37 @@ export async function bootstrapLocalTemplate(
   })
 
   // ...and a studio config (`sanity.config.[ts|js]`)
-  const studioConfig = await createStudioConfig({
+  const studioConfig = createStudioConfig({
     template: template.configTemplate,
     variables,
   })
 
   // ...and a CLI config (`sanity.cli.[ts|js]`)
-  const cliConfig = await createCliConfig({
-    projectId: variables.projectId,
-    dataset: variables.dataset,
-    autoUpdates: variables.autoUpdates,
-  })
+  const cliConfig = isStudioTemplate
```

Comment:
🤔 I am wondering if we should flip this logic, instead of checking `isStudioTemplate` which is basically checking the name and currently we allow users to build templates if someone builds `github.com/<id>/studio-core-app` it will categorize it as core app which is unintended behavior of creating a different kind of template. 

I think as is, we should flip it so that the template checks for a specific name `=== core-app` and currently user's can't make custom core apps. In future I was envisioning that we check the dependencies of the template and find the proper packages to confirm this which seems more stable but I don't know how feasible that is.

Reply:
Hmm, so the current logic to verify a studio template is in `determineStudioTemplate` (called further up in the file), which does check for the specific name `core-app`. Right now, the `github.com/<id>/studio-core-app` wouldn't get to this point in logic, because that is not a local template (there's a different file for remote templates).

Likewise, the `determineStudioTemplate` used in `initProject` also wouldn't fall into this because I believe you pass a URL. 

I'm not sure what would be best to add but happy to take suggestions.

Reply:
Per our convo yesterday, I've swapped the conditional -- hopefully it's more readable.

I'm still on the side of keeping creating core app templates fairly locked down for now. In the future we probably want a more robust way to let people make remote templates, or register templates. It's hard to determine how to architect that until we see which way the CLI is oging.